platformio.ini

```diff
@@ -3,6 +3,7 @@
 include_dir = .
 src_dir = ./src
 lib_dir = ./src/libs
+test_dir = ./unit_tests
```

Comment:
why not just use default "test" dir?

Reply:
Because we had talked about the possibility of having integration tests, and I wanted to set a precedent that this dir will only have unit test in it.